src/filesystem/implementation.js

```diff
@@ -1250,12 +1263,6 @@ function truncate_file(context, path, length, callback) {
       if(!fileData) {
         return callback(new Errors.EIO('Expected Buffer'));
       }
-      try {
```

Comment:
How come this got removed?  I wonder if this is your bug in Travis.

Reply:
I am almost sure I did not touch that part.

Reply:
OK, it needs to go back in.  Would you be able to add back this section and the other on line 42?  That will fix that error.